istioctl/cmd/root.go

```diff
@@ -299,3 +300,30 @@ func seeExperimentalCmd(name string) *cobra.Command {
 		},
 	}
 }
+
+var unsupportedCmdAlternatives = map[string]string{
+	"authz":          "none",
+	"bug-report":     "use istio-must-gather",
+	"dashboard":      "use Kiali",
+	"install":        "create Istio CR",
+	"kube-inject":    "set the `istio-injection=enabled` label",
+	"manifest":       "none",
+	"tag":            "use IstioRevisionTag CR",
+	"uninstall":      "delete Istio CR",
+	"upgrade":        "use IstioRevision CR",
+	"waypoint":       "none", //TODO: revisit once ambient mode is supported in Sail Operator.
+	"ztunnel-config": "none", //TODO: revisit once ambient mode is supported in Sail Operator.
+}
```

Comment:
Wouldn't it be better to have the alternative for each command specified as an argument to `unsupportedCmd`? Now we have to update the code in two places whenever we want to mark a command as unsupported, so it's easy to forget to add the alternative.

Reply:
Done